packages/core/src/Button/Button.tsx

```diff
@@ -144,17 +159,23 @@ const propTypes = {
   variation: PropTypes.oneOf(Object.keys(variations)),
   color: deprecatedColorValue(),
   disabled: PropTypes.bool,
+  ...boxShadow.propTypes,
+  borderRadius: PropTypes.oneOf(['', 'none', 'sm', 'md', 'lg']),
```

Comment:
Let's make sure these values are only defined once in this file. Also, You should be able to omit the `''` value since `borderRadius` isn't a required prop.